it/src/test/java/org/corfudb/universe/scenario/ScenarioUtils.java

```diff
@@ -104,4 +104,14 @@ static void waitForClusterDown(CorfuTable table) {
             log.info("Successfully waited failure detector to detect cluster down");
         }
     }
+
+    /**
+     * Wait for a specific amount of time. This should only be used when there is nothing
+     * else we can wait on, e.g. no layout change, no cluster status change.
+     *
+     * @param duration duration to wait in seconds
+     */
+    static void waitUninterruptibly(long duration) {
```

Comment:
Not sure if we can justify adding this one liner method. I'm for introducing simple methods for added readability but it doesn't seem that this one adds too much.

Reply:
Two reasons: 1) keep align with other wait methods, so every time we want to wait on sth., we use the methods in this file 2) the comments here acts as a warning not to use this waitUninterruptibly unless  there's nothing else to wait on.